src/connection-string-parser.ts

```diff
@@ -63,14 +63,22 @@ export class CodegenConnectionStringParser {
     try {
       void new URL(connectionString);
     } catch {
-      throw new SyntaxError(`Invalid URL: '${connectionString}'`);
+      if (!connectionString.includes('Server')) {
+        throw new SyntaxError(`Invalid URL: '${connectionString}'`);
+      }
+    }
+
+    let dialectName: CodegenDialectName = 'sqlite';
+    if (connectionString.includes('Server')) {
```

Comment:
A postgres url could easily contain the word `Server`. For example the username could be `dbServerUser`. This needs a better test.

Reply:
You're right. But a postgres would also start with the postgres protocol `postgres://`, to which there is a test in line 77